src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -1,13 +1,16 @@
 // The enum values in this file is taken from Blueprints and still has to match those values until we control ingest.
 export enum Tv2SourceLayer {
   CAMERA = 'studio0_camera',
+  REMOTE = 'studio0_live',
   JINGLE = 'studio0_jingle',
   AUDIO_BED = 'studio0_audio_bed',
   GRAPHICS_ACTION_COMMAND = 'studio0_adlib_graphic_cmd',
   AUDIO_ACTION_COMMAND = 'studio0_sisyfos_adlibs',
   DOWNSTREAM_KEYER_ACTION_COMMAND = 'studio0_dsk', // Original one from Blueprint includes the number of the DSK, followed by '_cmd'.
   DVE = 'dve',
-  REMOTE = 'studio0_live',
+  REPLAY = 'studio0_local',
+  REPLAY_STUDIO_AUX = 'studio0_aux_studio_screen',
+  REPLAY_VIZ_AUX = 'studio0_aux_viz_full1',
```

Comment:
Would be nice with the AUXs being expanded to `AUXILIARY`.

Reply:
Updated